client/components/Nav.jsx

```diff
@@ -146,6 +150,30 @@ class Nav extends React.PureComponent {
                   New
                 </button>
               </li>
+              <li className="nav__dropdown-item">
```

Comment:
I think these options should be under the "Sketch" dropdown and not the "File" dropdown, since these options are specific to the selected sketch.

Reply:
@catarak Alright!! :smile: Done the necessary changes in my next commit.